middleware/limiter/limiter_fixed.go

```diff
@@ -44,6 +44,9 @@ func (FixedWindow) New(cfg Config) fiber.Handler {
 
 		// Lock entry
 		mux.Lock()
+		defer func() {
+			mux.Unlock()
```

Comment:
this is not a good idea, the mutex in the defer would block the reading for too long

Because with c.Next() all possible handlers are executed